lib/ast.js

```diff
@@ -1418,6 +1418,34 @@ var AST_This = DEFNODE("This", null, {
     },
 }, AST_Symbol);
 
+var AST_Template = DEFNODE("Template", "expressions strings tag", {
+    $documentation: "A template literal, i.e. tag`str1${expr1}...strN${exprN}strN+1`",
+    $propdoc: {
+        expressions: "[AST_Node*] the placeholder expressions",
+        strings: "[string*] the interpolating text segments",
```

Comment:
Might be good to mention `strings` are in raw form.

Reply:
Done in https://github.com/mishoo/UglifyJS/pull/4603/commits/c830049e0127cc083574545a80ca5ff8866f03ef